db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,36 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :deployment_authentications do |t|
+      t.string :name
+      t.boolean :challenge
+      t.boolean :login
+      t.string :kind
+      t.text :htpassd_users
+      t.text :ldap_id, :array => true, :default => []
+      t.text :ldap_email, :array => true, :default => []
+      t.text :ldap_name, :array => true, :default => []
+      t.text :ldap_preferred_user_name, array: true, :default => []
+      t.string :ldap_bind_dn
+      t.string :ldap_bind_password
+      t.string :ldap_ca
```

Comment:
@alongoldboim yes I know (the name was self-explanatory) but I believe that in `openshift-ansible` that's a path to a file. Who is delivering that file? What's the content? What do you store here?